catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryLuceneDirectoryIndex.java

```diff
@@ -94,9 +95,14 @@ protected IndexSearcher createIndexSearcher(final IndexReader indexReader) {
   @Override
   public List<GeoEntry> query(final String queryString, final int maxResults)
       throws GeoEntryQueryException {
-    final Directory directory = openDirectoryAndCheckForIndex();
 
-    return doQuery(queryString, maxResults, directory);
+    try (final Directory directory = openDirectoryAndCheckForIndex()) {
+      return doQuery(queryString, maxResults, directory);
+    } catch (IOException e) {
+      LOGGER.warn("Error querying from directory", e);
```

Comment:
✏️  We might want to trace the exception stack trace at debug and not at warn level